images/nginx/rootfs/build.sh

```diff
@@ -465,7 +465,7 @@ make install
 
 # Get Brotli source and deps
 cd "$BUILD_PATH"
-git clone --depth=100 https://github.com/google/ngx_brotli.git
+git clone --depth=1 https://github.com/google/ngx_brotli.git
 cd ngx_brotli
 # https://github.com/google/ngx_brotli/issues/156
 git reset --hard 63ca02abdcf79c9e788d2eedcc388d2335902e52
```

Comment:
Due to the previous modification being reverted, 
https://github.com/kubernetes/ingress-nginx/pull/10697
this fix still exists here.

Can you remove lines 470-471 so that we can use `--depth=1` 

Thanks!